### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by teiling88**
removing duplicates

### CODE REVIEW

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 1]
```diff
@@ -240,8 +239,6 @@ For this operation we recommend the console command `sw:thumbnail:generate` to a
     * `s_core_auth.admin`
     * `s_core_auth.salted`
     * `s_order_basket.liveshoppingID`
-    * `s_order_basket.liveshoppingID`
```
Comment (bcremer) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Good catch :+1:

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 2]
```diff
@@ -124,7 +124,6 @@ For this operation we recommend the console command `sw:thumbnail:generate` to a
 * Template inheritance using `{extends file="[default]backend/..."}` is no longer supported and should be replaced by `{extends file="parent:backend/..."}`
 * Removed smarty variable `$sArticle.sNavigation` in product detail page
 * Removed support for flash banners. The associated template block `frontend_listing_swf_banner` is marked as deprecated
-* Removed the template files for the feed functionality, which was marked as deprecated in SW 3.5
```
Comment (bcremer) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Why did you remove that line?

Comment (teiling88) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: image

**Comment by bcremer** [comment]
Thank you @teiling88 will be deployed soon.
